.github/workflows/base.yml

```diff
@@ -34,8 +34,6 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-${{ steps.nvm.ouputs.NVMRC }}-nodemodules-${{ hashFiles('**/yarn.lock') }}
             ${{ runner.os }}-${{ steps.nvm.ouputs.NVMRC }}-nodemodules-
-      - name: Authenticate with NPM_TOKEN for private NPM packages
-        run: echo "//registry.npmjs.org/:_authToken=${{ secrets.NPM_TOKEN }}" > ~/.npmrc
```

Comment:
What do we need to do to remove `NPM_TOKEN` from secrets?

Reply:
It was added to this repo's [secrets](https://docs.github.com/en/actions/security-guides/encrypted-secrets), I'll get in touch with IT to remove it after this is merged.